### DISCUSSION TITLE
Does anyone have llama-65b throughput perf data on multi A100 40GB?

### DISCUSSION THREAD

**Discussion by zhaoyang-star** [vllm-project/vllm#443] [Q&A]
Does anyone have llama-65b throughput perf data on multi A100 40GB?

**Comment by irasin** [comment]
Here's the perf data of llama65B on A100 80GB, refer to https://github.com/vllm-project/vllm/issues/421, hope to help.

**Reply by zhaoyang-star** [reply]
Thanks. I had a glace at #421 , which is a very solid work. I will go deeper in the next few days.
